# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/rene/emacs-xemacs-shared/no-shared.patch
# ROCK Linux is Copyright (C) 1998 - 2003 Clifford Wolf
# 
# This program is free software; you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation; either version 2 of the License, or
# (at your option) any later version. A copy of the GNU General Public
# License can be found at Documentation/COPYING.
# 
# Many people helped and are helping developing ROCK Linux. Please
# have a look at http://www.rocklinux.org/ and the Documentation/TEAM
# file for details.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- xemacs-21.4.12/lib-src/Makefile.in.in	2002-08-20 13:34:16.000000000 +0200
+++ xemacs-21.4.12-fixed/lib-src/Makefile.in.in	2003-03-29 03:20:01.000000000 +0100
@@ -238,19 +238,7 @@
 ## Install the internal utilities.  Until they are installed, we can
 ## just run them directly from lib-src.
 ${archlibdir}: all
-	@echo; echo "Installing utilities run internally by XEmacs."
-	./make-path ${archlibdir}
-	if test "`(cd ${archlibdir} && $(pwd))`" != "`$(pwd)`"; then \
-	  for f in ${PRIVATE_INSTALLABLE_EXES}; do \
-	    (cd .. && $(INSTALL_PROGRAM) lib-src/$$f ${archlibdir}/$$f) ; \
-	  done ; \
-	fi
-	if test "`(cd ${archlibdir} && $(pwd))`" \
-	     != "`(cd ${srcdir}     && $(pwd))`"; then \
-	  for f in ${PRIVATE_INSTALLABLE_SCRIPTS}; do \
-	    (cd .. && $(INSTALL_PROGRAM) ${srcdir}/$$f ${archlibdir}/$$f); \
-	  done ; \
-	fi
+	@echo; echo "Not installing utilities run internally by XEmacs."
 
 install: ${archlibdir}
 	@echo; echo "Installing utilities for users to run."
